src/components/AddToList.tsx

```diff
@@ -1,9 +1,10 @@
 import React, { useState } from 'react'
-import { IState as Props } from "../App";
+import {IPerson} from "../models/IPerson";
+
 
 interface IProps {
-    setPeople: React.Dispatch<React.SetStateAction<Props["people"]>>
-    people: Props["people"]
+    setPeople: React.Dispatch<React.SetStateAction<IPerson[]>>
```

Comment:
Using `React.Dispatch<React.SetStateAction<>>` already introduces coupling as we are basically saying that it must be a native react state setter the needs to be passed in here. But what if we are not using react state in the parent component?

We could, instead, define what the `AddToList` component actually expects: A function that takes a list of persons and returns nothing:
```suggestion
    setPeople: (persons: IPerson[]) => void
```